src/components/timer.js

```diff
@@ -63,14 +65,15 @@ class Timer extends Component {
   };
 
   render() {
-    const { mode, time } = this.state;
+    const { mode, time, pomodoroCount } = this.state;
 
     return (
       <div className="timer-container">
         <div className="timer-mode">{mode}</div>
         <div className="timer-time">
           {Math.floor(time / 60)}:{(time % 60).toLocaleString('en-US', { minimumIntegerDigits: 2 })}
         </div>
+        <div className="pomodoro-counter">Pomodoros completed: {pomodoroCount}</div>
```

Comment:
should be 'completed pomodoros' instead of 'Pomodoros completed'

Reply:
before i can approve the pr merge this branch with main.

Reply:
Changes are now pushed! Waiting 4 a review on them 👍🏼